### PR TITLE
[openblas] Fix C/CXX flag customization in openblas

### PR DESCRIPTION
**Pull request by ahojnnes**
The previous code was broken when VCPKG_C_FLAGS or VCPKG_CXX_FLAGS were already defined in the triplet config file.

### CODE REVIEW

File: ports/openblas/portfile.cmake [hunk 1]
```diff
@@ -78,8 +78,8 @@ elseif(VCPKG_TARGET_IS_WINDOWS)
             ${COMMON_OPTIONS}
     )
 else()
-    list(APPEND VCPKG_C_FLAGS "-DNEEDBUNDERSCORE") # Required to get common BLASFUNC to append extra _
-    list(APPEND VCPKG_CXX_FLAGS "-DNEEDBUNDERSCORE")
+    set(VCPKG_C_FLAGS "${VCPKG_C_FLAGS} -DNEEDBUNDERSCORE") # Required to get common BLASFUNC to append extra _
+    set(VCPKG_CXX_FLAGS "${VCPKG_CXX_FLAGS} -DNEEDBUNDERSCORE")
```
Comment (Neumann-A) [inline ports/openblas/portfile.cmake:82]:
```suggestion
    string(APPEND VCPKG_C_FLAGS " -DNEEDBUNDERSCORE") # Required to get common BLASFUNC to append extra _
    string(APPEND VCPKG_CXX_FLAGS " -DNEEDBUNDERSCORE")
```

**Review by JackBoosY — CHANGES_REQUESTED** [review]
Please bump the port version. See [documentation](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#update-the-version-field-in-the-control-file-of-any-modified-ports).
Also please run command `vcpkg x-add-version --all` then commit the changes.

**Comment by ahojnnes** [comment]
@JackBoosY Updated the versions. Thanks.

File: versions/o-/openblas.json [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "d8dce7920450b773b32512fb5a2b319c9449d3fe",
```
Comment (JackBoosY) [inline versions/o-/openblas.json:4]:
```suggestion
      "git-tree": "120f54c302949aa0b6c6926eed2bf9b9ee7d8574",
```

Comment (JackBoosY) [reply versions/o-/openblas.json:4]: Please wait for the osx test finish then commit suggestion.

**Review by JackBoosY — APPROVED** [review]

**Comment by JackBoosY** [comment]
Can you please resolve the file conflict?

Thanks.

**Comment by JackBoosY** [comment]
Can you please resolve the file conflict?

Thanks.

**Comment by ahojnnes** [comment]
@JackBoosY Done. Would appreciate to get this merged to avoid another conflict. Thanks!

File: versions/o-/openblas.json [hunk 3]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "eafe66df0d415ab1a0b10399bdc391d6e043fd22",
```
Comment (JackBoosY) [inline versions/o-/openblas.json:4]:
```suggestion
      "git-tree": "20d57360e6e7afa4e4f033a87e1ded91571ee462",
```

**Review by JackBoosY — APPROVED** [review]

**Pull request merged**